docs/reference/sql/functions/conditional.asciidoc

```diff
@@ -155,3 +155,38 @@ include-tagged::{sql-specs}/docs.csv-spec[nvlReturnFirst]
 ----
 include-tagged::{sql-specs}/docs.csv-spec[nvlReturnSecond]
 ----
+
+
+[[sql-functions-conditional-nullif]]
+==== `NULLIF`
+
+.Synopsis
+[source, sql]
+----
+NULLIF ( expression<1>, expression<2> )
+----
+
+*Input*:
+
+<1> 1st expression
+
+<2> 2nd expression
+
+
+*Output*: `NULL` if the 2 expressions are equal, otherwise the 1st expression.
```

Comment:
Lowercase `null`? For another function doc page you used the lowercased version of `null`: https://github.com/elastic/elasticsearch/pull/35793/files#diff-cf102e856ee9e39135c9677a52db35e1R108